src/codelabs/async-await/index.md

```diff
@@ -176,23 +177,27 @@ when using `async` and `await`:
 * __The `await` keyword works only in `async` functions.__
 * __To define an async function, add `async` before the function body:__
```

Comment:
Remove colon(:)?

Reply:
Oh, I noticed that @kwalrath suggested to add this colon. Hi Kathy, it's a little confusing to me to have this colon at the end of a list <ul>. Also, the content below this list seem to include two points.

Reply:
The colon isn't supposed to be in the list, it's supposed to be in the intro to the sample.